hosting/forms.py

```diff
@@ -20,9 +26,11 @@ def clean(self):
         password = self.cleaned_data.get('password')
         is_auth = authenticate(email=email, password=password)
         if not is_auth:
-            raise forms.ValidationError("Your username and/or password were incorrect.")
+            raise forms.ValidationError(
```

Comment:
Can we add the translation function for the text
"Your username and/or password were incorrect." ?